src/components/AvatarGroup/AvatarGroupCounterTooltipContainer.tsx

```diff
@@ -50,7 +72,7 @@ const AvatarGroupCounterTooltipContainer = ({
   const hideTrigger = useMemo(() => [Dialog.hideShowTriggers.MOUSE_LEAVE], []);
 
   if (!avatars?.length && !counterTooltipCustomProps?.content) {
-    return children;
+    return children as ReactElement<CounterProps>;
```

Comment:
why it's suddenly needed?

Reply:
There were a type collision cause of that, fixed it in a different way https://github.com/mondaycom/monday-ui-react-core/pull/931/commits/e19b1c8117f5d2cee45eb1b33f79302921506bf6